### PR TITLE
explorer: Add function to return node URL

### LINKED ISSUE #2588: explorer: Update node URL creation process
In order to work with a local node, the GQL queires need to use a function that returns the URL for them to connect to based off the selected network and use it's own execution environment's protocol to determine the connection URL to the node.

### PR DESCRIPTION
**Pull request by kieranhall**
Resolves #2588

### CODE REVIEW

File: explorer/src/lib/dusk/svelte-stores/createPollingDataStore.js [hunk 1]
```diff
@@ -48,14 +48,16 @@ const createPollingDataStore = (dataRetriever, fetchInterval) => {
   };
 
   const stop = () => {
-    document.removeEventListener("visibilitychange", visibilityChangeHandler);
+    document &&
```
Comment (ascartabelli) [inline explorer/src/lib/dusk/svelte-stores/createPollingDataStore.js:51]: How is it possible for the document to not exist?

**Comment by nortonandreev** [comment]
There's a typo in the issue ID.

**Comment by kieranhall** [comment]
> There's a typo in the issue ID.

Time for me to dig out my glasses.

**Comment by kieranhall** [comment]
Made the changes inline with @ZER0's comments.  Went to the point of removing the `network` and  `networks` from the appStore, but that will be a separate PR, as it involves a lot more refactoring.

**Review by deuch13 — APPROVED** [review]

**Review by deuch13 — APPROVED** [review]
LGTM!

**Review by deuch13 — APPROVED** [review]

**Pull request merged**